km/km_coredump.c

```diff
@@ -586,6 +588,24 @@ static inline int km_core_write_notes(km_vcpu_t* vcpu,
    return 0;
 }
 
+static inline int km_corefile_align(int corefd, off_t* offp)
+{
+   off_t off = lseek(corefd, 0, SEEK_CUR);
```

Comment:
This is a question not a comment. 
If the off is unaligned we are skipping the current page. Is this what we want?